Soften home server capacity claim and remove specific example (#1134)
jamulussoftware/jamulussoftware.github.io: wiki/en/Running-a-Server.md

Usually, problems are on the _Client_ side and should be fixed there. Have a look at the [Troubleshooting page](/wiki/Client-Troubleshooting) if needed.

> However, various problems can also arise when setting up Servers - especially when run on a low-bandwidth home connection. It can work fine to have less than 5 players on a slower-speed home connection. You can read more about network requirements at [different quality settings here](Server-Bandwidth).

Author: _can_, maybe _might_ should appear here. I see hosting at home as the Achilles Heel of the whole technology. Many people start there, thinking they really want or need to, and many of those people have a bad experience, due to upstream speed or even wireless residential bottlenecks.

Reviewer: I tend to agree -- we shouldn't really suggest less than 2Mbps each way per client.  With 4.0.0, we'll need to up that, too - the Max Quality work will need to make it clear what the Server needs.

Author: The introduction of wireless residential seems to throw a lot of the traditional throughput arithmetic out the window. We're conflating _capacity_ with _consistent timeliness_ and in my experience this breaks down often in residential scenarios. We might suggest avoiding hosting a server on a wireless connection altogether, and we might suggest that fiber is fine for hosting at home.

Reviewer: Yes - agreed: _hosting_ on wireless ... words fail me.

> However, various problems can also arise when setting up Servers - especially when run on a low-bandwidth home connection. It can work fine to have less than 5 players on a slower-speed home connection. You can read more about network requirements at [different quality settings here](Server-Bandwidth).

Author: ```suggestion
However, various problems can also arise when setting up Servers - especially when run on a low-bandwidth home connection. It can work fine to have less than 5 players on a slower-speed home connection. A wireless internet connection is not recommended for a server in your home, while a fiber connection should work well. You can read more about network requirements at [different quality settings here](Server-Bandwidth).
```

> However, various problems can also arise when setting up Servers - especially when run on a low-bandwidth home connection. It can work fine to have up to 5 players on a slower-speed home connection. A wireless internet connection is not discouraged for a server in your home, while a fiber connection should work well. You can read more about network requirements at [different quality settings here](Server-Bandwidth).

Author: ```suggestion
However, various problems can also arise when setting up Servers - especially when run on a low-bandwidth home connection. It can work fine to have up to 5 players on a slower-speed home connection. A wireless internet connection is discouraged for a server in your home, while a fiber connection should work well. You can read more about network requirements at [different quality settings here](Server-Bandwidth).
```

> However, various problems can also arise when setting up Servers - especially when run on a low-bandwidth home connection. It can work fine to have less than 5 players on a slower-speed home connection. A wireless internet connection is not recommended for a server in your home, while a fiber connection should work well. You can read more about network requirements at [different quality settings here](Server-Bandwidth).

Reviewer: "less than" -> "no more than" -- a bit "harder".

"is not recommended" -- "is discouraged".

The Server Bandwidth page _should_ explain about ping, latency and the jitter buffers...

> However, various problems can also arise when setting up Servers - especially when run on a low-bandwidth home connection. It can work fine to have less than 5 players on a slower-speed home connection. A wireless internet connection is not recommended for a server in your home, while a fiber connection should work well. You can read more about network requirements at [different quality settings here](Server-Bandwidth).

Author: ```suggestion
However, various problems can also arise when setting up Servers - especially when run on a low-bandwidth home connection. It can work fine to have up to 5 players on a slower-speed home connection. A wireless internet connection is not discouraged for a server in your home, while a fiber connection should work well. You can read more about network requirements at [different quality settings here](Server-Bandwidth).
```
